docs/getting-started/quickstart.md

```diff
@@ -35,10 +26,98 @@ export NVM_DIR="$HOME/.nvm"
+
+Choose a database instance name, username and password. Both the instance name and username must be 3 to 16 characters long and contain only lowercase letters, numbers and underscores. The password must contain at least 8 characters. Run this command and enter the password when prompted:
+```
+npx dbos-cloud db provision <database-instance-name> -U <database-username>
+```
+
+If successful, the command should print `Database successfully provisioned!`. To view your provisioned instance, run `npx dbos-cloud db list`. For more information on cloud database management, check out [our guide](../cloud-tutorials/database-management.md).
+
+:::info
+This instance can host multiple independent databases for different applications.
+By default, the "Hello" app uses a database named `hello`. This is configurable in [`dbos-config.yaml`](../api-reference/configuration.md)
+:::
+
+### 5. Deploy!
+
+Now, we're ready to deploy your application to DBOS Cloud! First, register your application by running this command, using your database instance name from the last step:
+
+```
+npx dbos-cloud app register -d <database-instance-name>
+```
+
+If successful, the command should print `Successfully registered <app-name>!`
+
+Finally, deploy your application to run it in the cloud:
+
+```
+npx dbos-cloud app deploy
+```
+:::tip
+The deploy command may take a minute or so to finish
+:::
+
+If successful, the command will print `Successfully deployed <app-name>! Access your application at <URL>`. To see that your app is working, visit `<URL>/greeting/dbos` in your browser. For example, if your username is `mike` and your app name is `hello`, you would visit: 
+```
+https://mike-hello.cloud.dbos.dev/greeting/dbos
+```
+
+:::tip
+Don't forget to add `/greeting/dbos` at the end of the URL. 
+:::
+
+Each time you refresh the page, the counter will go up by one. If you change the last part of the URL from `dbos` to another name like `/greeting/alice`, the counter will start at 1 for each new name.
 
-You can install Postgres locally or launch it in a Docker container.
+Congratulations, you've successfully deployed your first application to DBOS Cloud! You can see your deployed app in the [cloud console](https://console.dbos.dev/) or in the cli by running `npx dbos-cloud app status`.
 
+## Run the App Locally
+
+For development, debugging, testing, or a self-hosted deployment, here's how to run this app on your local machine. This section assumes you've already created an application folder as described in the last secion.
+
+### 1. Install Postgres or use Docker
+
+The app needs a [PostgreSQL](https://www.postgresql.org/) database to connect to. You can install Postgres on your system or launch it in a Docker container:
+
+<Tabs groupId="operating-systems">
+<TabItem value="mac" label="macOS">
```

Comment:
Switch the order of the tabs here, so Postgres/Docker is the outer tab and OS is the inner one.

Reply:
Done